avogadro/rendering/glrenderer.cpp

```diff
@@ -27,8 +27,25 @@ using Core::Array;
 GLRenderer::GLRenderer()
   : m_valid(false), m_textRenderStrategy(nullptr), m_center(Vector3f::Zero()),
     m_radius(20.0)
+#ifdef _3DCONNEXION
+    ,
+    m_drawIcon(false), m_iconData(nullptr), m_iconWidth(0u), m_iconHeight(0u),
+    m_iconPosition(Eigen::Vector3f::Zero())
+#endif
 {
   m_overlayCamera.setIdentity();
+#ifdef _3DCONNEXION
```

Comment:
I would be willing to see how this works in general. It doesn't seem specific to your code.

Reply:
Ok, I will remove mentioned #ifdef's from GLRenderer making my changes to this class general.